test/testtokenize.cpp

```diff
@@ -4700,6 +4700,16 @@ class TestTokenizer : public TestFixture {
             ASSERT_EQUALS(true, tok1->link() == tok2);
             ASSERT_EQUALS(true, tok2->link() == tok1);
         }
+        {
+            // #8654
+            const char code[] = "template<int... Ns> struct foo : A<Ns>... {};";
```

Comment:
I assume due to the low-level test that incomplete test code is appropriate?

Reply:
I think just prepending `template<int N> struct A {}` to the test will make it more complete.

Reply:
Will make it more complete in the next iteration. Thanks.